pkg/test/resourcefixture/testdata/basic/redis/v1beta1/basicrediscluster/_generated_object_basicrediscluster.golden.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: redis.cnrm.cloud.google.com/v1alpha1
+kind: RedisCluster
+metadata:
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: rediscluster-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  location: us-central1
+  projectRef:
+    external: ${projectId}
+  pscConfigs:
+  - networkRef:
+      external: projects/${projectId}/global/networks/computenetwork-${uniqueId}
+  shardCount: 10
+status:
+  conditions:
+  - lastTransitionTime: "1970-01-01T00:00:00Z"
+    message: The resource is up to date
+    reason: UpToDate
+    status: "True"
+    type: Ready
+  observedGeneration: 2
+  observedState:
+    createTime: "1970-01-01T00:00:00Z"
+    discoveryEndpoints:
+    - address: 10.128.0.3
+      port: 6379
+      pscConfig:
+        networkRef:
+          external: projects/${projectId}/global/networks/computenetwork-${uniqueId}
```

Comment:
Not a blocker (but just a marker)
We want to flatten the `*Ref.external` style in `status`. The CRD shall be changed (since it's alpha) to `network: projects/${projectId}/global/networks/computenetwork-${uniqueId}`

Reply:
Actually, do we want to add the `status.observedState`? @justinsb